ci-matrix.py

```diff
@@ -38,15 +38,19 @@ def parse_labels(labels: Optional[str]) -> dict[str, set[str]]:
     for label in labels.split(","):
         label = label.strip()
 
-        # Handle special directive labels
-        if label in CI_SKIP_LABELS:
+        # Handle special labels
+        if label in CI_EXTRA_SKIP_LABELS:
             result["directives"].add("skip")
             continue
 
         if not label or ":" not in label:
             continue
 
         category, value = label.split(":", 1)
+
+        if category == "ci":
+            category = "directives"
```

Comment:
I could just avoid this rename and use `ci` instead of `directives` throughout, not sure if it really matters.